experiments/results/plots_data/canonical-tracr-proportion-l2-False-0.json

```diff
@@ -20,11 +13,18 @@
             ],
             "test_kl_div": [
               Infinity,
-              0.16910363733768463,
-              0.004418146796524525,
+              -6.007197517199359e-10,
+              -6.007197517199359e-10,
```

Comment:
As far as I can tell this is replacing non zero entries with zero entries which shows correctness